src/components/common/footer/footer.jsx

```diff
@@ -1,5 +1,6 @@
 import './footer.css'
 function Footer() {
+  console.log('footer is called')
```

Comment:
why you added this?

Reply:
To check that how many time footer is called.